### ISSUE TITLE
Question: name as attribute with serde

### ISSUE THREAD

**Issue by WesleyAC** [tafia/quick-xml#715]
I have a struct like:

```rust
struct Foo {
    one: String,
    two: String,
}
```

and I want XML like:

```xml
<foo>
    <item name="one">value of string 1</item>
    <item name="two">value of string 2</item>
</foo>
```

I don't see a easy way to do this kind of thing with quick-xml's serde option — my experiments have been using `PhantomData` to create a `Item<K, V>` type where the `K` is a `PhantomData<K>` with a custom trait to hold the name, and then I use the serde `with` attribute, but that feels quite messy and annoying. Is there a easy way to represent this? (I wouldn't choose to use XML like this, but the API I'm using does, unfortunately...)

**Comment by Mingun** [comment]
Yes, [you should start](https://docs.rs/quick-xml/latest/quick_xml/de/index.html#basics) attribute fields from `@` using `#[serde(rename = "@...")]`.

**Comment by Mingun** [comment]
There is no built-in mappings for such transformations. Probably, your way to do that is the right approach. If you post your solution here, maybe we could help you to find the ways to optimize it.
